dlt/pipeline/pipeline.py

```diff
@@ -466,7 +471,7 @@ def run(
         # extract from the source
         if data is not None:
             self.extract(data, table_name=table_name, write_disposition=write_disposition, columns=columns, primary_key=primary_key, schema=schema)
-            self.normalize(loader_file_format=loader_file_format)
+            self.normalize(loader_file_format=loader_file_format, schema_contract_settings=schema_contract_settings)
```

Comment:
it looks like we should pass it to extract stage, not normalize. and apply to all sources being extracted there.
do not send it to normalize - it will afffect all the schemas - not only those from sources actually extracted

Reply:
I moved it, but am not quite sure if it is in the right location, I think so though.